crates/bevy_pbr/src/light/mod.rs

```diff
@@ -1825,12 +1826,12 @@ pub fn update_spot_light_frusta(
         // by applying those as a view transform to shadow map rendering of objects
         let view_backward = transform.back();
 
-        let spot_view = spot_light_view_matrix(transform);
-        let spot_projection = spot_light_projection_matrix(spot_light.outer_angle);
-        let view_projection = spot_projection * spot_view.inverse();
+        let spot_view_from_world = spot_light_view_from_world(transform);
```

Comment:
As noted, this function should be called world from view and so should the variable. I noticed this because the calculation below didn’t make sense with the new naming.